functions/controllers/user/signUp.js

```diff
@@ -10,6 +12,8 @@ const {
 const { validateSignUpData } = require('../../util/validator');
 const { configConstants, status, message } = require('../../util/constants');
 
+dotenv.config();
```

Comment:
why are you loading the env files again. Only load is once at the most top level `index.js` file